newrelic-api-client/src/main/java/com/ocadotechnology/newrelic/apiclient/model/conditions/nrql/AlertsNrqlCondition.java

```diff
@@ -0,0 +1,36 @@
+package com.ocadotechnology.newrelic.apiclient.model.conditions.nrql;
```

Comment:
Shouldn't it be in `package com.ocadotechnology.newrelic.apiclient.model.conditions` package?

Reply:
`AlertsExternalServiceCondition` is in separate `package com.ocadotechnology.newrelic.apiclient.model.conditions.external` package